packages/pharos/src/components/alert/pharos-alert.ts

```diff
@@ -98,6 +125,7 @@ export class PharosAlert extends FocusMixin(LitElement) {
         <div class="alert__body">
           <slot @slotchange=${this._handleSlotChange}></slot>
         </div>
+        ${this._getCloseButton()}
```

Comment:
I think we've been using the `_render*` convention for functions that return HTML